eZ/Publish/API/Repository/Values/User/Policy.php

```diff
@@ -56,6 +57,14 @@
     protected $function;
 
     /**
+     * Original policy ID the policy was created from.
+     * Mostly used when role status is Role::STATUS_DRAFT.
+     *
+     * @var mixed
+     */
+    protected $originalId;
```

Comment:
This does not really belong here, it should ideally be a PolicyDraft, and return that from API whenever this id is set from db (so most likely need to use -1 for new draft).